pkg/moov/transfer_models.go

```diff
@@ -126,6 +128,26 @@ type Transfer struct {
 	SalesTaxAmount *Amount `json:"salesTaxAmount,omitempty"`
 }
 
+// Fees charged to accounts involved in the transfer.
+type MoovFees struct {
+	// ID of the account that fees were charged to.
+	AccountID string `json:"accountID,omitempty"`
+	// List of fee IDs that sum to the totalAmount.
+	FeeIDs []string `json:"feeIDs,omitempty"`
+	// The total amount of fees charged to the account.
+	TotalAmount TotalAmount `json:"totalAmount,omitempty"`
+	// Indicates whether the account charged was the partner, source, or destination of the transfer.
+	TransferParty string `json:"transferParty,omitempty"`
+}
+
+// The total amount of fees charged to the account.
+type TotalAmount struct {
+	// A 3-letter ISO 4217 currency code.
+	Currency string `json:"currency,omitempty"`
+	// A decimal-formatted numerical string that represents up to 9 decimal place precision.
+	ValueDecimal string `json:"valueDecimal,omitempty"`
+}
```

Comment:
```suggestion
// AmountDecimal A decimal value representing money in a specific currency.
type AmountDecimal struct {
	// A 3-letter ISO 4217 currency code.
	Currency string `json:"currency"`
	// A decimal-formatted numerical string that represents up to 9 decimal place precision. For example, $12.987654321 is '12.987654321'.
	ValueDecimal string `json:"valueDecimal"`
}
```

Suggestion to mainly updating naming / json tags

Reply:
Thanks for these suggestions. I was basing this off of the docs. Are those outdated? https://docs.moov.io/api/money-movement/transfers/list/